start.js

```diff
@@ -86,13 +86,14 @@ function runFastify (args, cb) {
 
   loadModules(opts)
 
-  var file = null
-  try {
-    file = require(path.resolve(process.cwd(), opts._[0]))
-  } catch (e) {
-    return module.exports.stop(e)
+  const filePath = path.resolve(process.cwd(), opts._[0])
+
+  if (!fs.existsSync(filePath)) {
+    return module.exports.stop(`${opts._[0]} doesn't exist within ${process.cwd()}`)
```

Comment:
```suggestion
    return module.exports.stop(new Error(`${opts._[0]} doesn't exist within ${process.cwd()}`))
```
I’m sorry for my previous suggestion. There must be an instance of Error, So that the test case can capture `error.message`